### PR TITLE
Remove pg_dump dependency

### PR DESCRIPTION
**Pull request by yudppp**
I checked `Drop pg_dump dependency` issue. https://github.com/k0kubun/sqldef/issues/10.
Easy to implement `Drop pg_dump dependency` and `Support SERIAL` together.

@k0kubun 
Please check my plan is correct.

## TODO
- Support SERIAL -> I will fix other PR(this PR is `Drop pg_dump dependenc` only)
- [x] Add test code
- [x] Support multi primary key

### CODE REVIEW

**Comment by k0kubun** [comment]
Wow, that looks great. I think it's okay to go with this once the existing integration tests pass. I appreciate your effort working on this.

**Comment by yudppp** [comment]
@k0kubun 
I Finished.And passed all existing integration tests

**Comment by k0kubun** [comment]
Sounds good. Thanks a lot!

File: adapter/postgres/postgres.go [hunk 1]
```diff
@@ -47,103 +48,241 @@ func (d *PostgresDatabase) TableNames() ([]string, error) {
...
+			}
+		}
+		col.Name = strings.Trim(colName, `" `)
+		if colDefault != nil || isPK {
+			if isPK {
+				col.IsPrimaryKey = true
+			} else {
+				col.Default = *colDefault
+			}
+		}
```
Comment (k0kubun) [inline adapter/postgres/postgres.go:174]: Just curious, did you have any reason why this wasn't:

```go
		if isPK {
			col.IsPrimaryKey = true
		}
		if colDefault != nil {
			col.Default = *colDefault
		}
```
More specifically, did you intend to skip setting `col.Default` when `colDefault != nil` and `isPk` are true?

**Pull request merged**